### PR TITLE
Greatandamanese

### PR DESCRIPTION
**Pull request by d97hah**
Still no line wrap on cfg.set but no biggie...

### CODE REVIEW

**Comment by xrotwang** [comment]
There's almost only deletions in this pr. Inteded or forgot to add files?
Am 12.03.2017 18:21 schrieb "d97hah" <notifications@github.com>:

**Comment by d97hah** [comment]
thx yes missed add, now added that, does it work?

File: languoids/tree/grea1241/md.ini [hunk 1]
```diff
@@ -6,9 +6,10 @@ level = family
 [classification]
 sub = **hh:hv:Manoharan:Andamanese**
 familyrefs = 
-	**hh:hv:Abbi:Andamanese**
+	**hh:hv:Abbi:Andamanese** **hh:typ:ComrieZamponi:Great-Andamanese**
```
Comment (xrotwang) [inline languoids/tree/grea1241/md.ini]: Note: The convention for lists is to put each item in a separate line. So this should be
```
        **hh:hv:Abbi:Andamanese**
        **hh:typ:ComrieZamponi:Great-Andamanese**
```

File: languoids/tree/grea1241/md.ini [hunk 2]
```diff
@@ -6,9 +6,10 @@ level = family
...
 subrefs = 
 	**hh:hv:Manoharan:Andamanese**
+family =
```
Comment (xrotwang) [inline languoids/tree/grea1241/md.ini]: I tried to remove fields with empty values. Did you add this manually or did it sneak in as serialization of the classification object? If the latter, I'll have to fix this.

File: languoids/tree/grea1241/midd1355/midd1323/juwo1236/akak1253/md.ini [hunk 3]
```diff
@@ -30,7 +30,8 @@ lexvo =
 multitree = aky
 
 [classification]
-sub = **hh:hv:Manoharan:Andamanese**
+sub = **hh:hv:Manoharan:Andamanese** **hh:typ:ComrieZamponi:Great-Andamanese**
 subrefs = 
-	**hh:hv:Manoharan:Andamanese**
+	hh:hv:Manoharan:Andamanese
+	hh:typ:ComrieZamponi:Great-Andamanese
```
Comment (xrotwang) [inline languoids/tree/grea1241/midd1355/midd1323/juwo1236/akak1253/md.ini]: When I unified the way sources are referenced from INI files, I adopted the convention to **always** mark source keys with `**<key>**`. Btw. this is something that `glottolog check tree` should catch.

File: languoids/tree/grea1241/md.ini (hunk 1 above)
Comment (d97hah) [inline languoids/tree/grea1241/md.ini]: Ok it's the result of l.cfg.set('classification', 'familyrefs', ["**hh:hv:Abbi:Andamanese**", "**hh:typ:ComrieZamponi:Great-Andamanese**"]), should if be done elsehow or maybe I have and unupdated pyglottolog?!

File: languoids/tree/grea1241/md.ini (hunk 2 above)
Comment (d97hah) [inline languoids/tree/grea1241/md.ini]: It's from l.cfg.set('classification', 'family', "")!?

File: languoids/tree/grea1241/md.ini (hunk 1 above)
Comment (d97hah) [inline languoids/tree/grea1241/md.ini]: Thanks, fixed.

File: languoids/tree/grea1241/md.ini (hunk 2 above)
Comment (xrotwang) [inline languoids/tree/grea1241/md.ini]: Ah, ok. For removal, I recommend
```python
del l.cfg['classification']['family']
```

File: languoids/tree/grea1241/md.ini (hunk 1 above)
Comment (xrotwang) [inline languoids/tree/grea1241/md.ini]: Hm. If `set` receives a `list` as value, it should format the value such that each item is on a separate line. This is done by code in `clldutils.inifile`. Considering that the first value is on a new line, I'd guess it was created by
```python
 l.cfg.set('classification', 'familyrefs', ["**hh:hv:Abbi:Andamanese** **hh:typ:ComrieZamponi:Great-Andamanese**"])
```

**Comment by d97hah** [comment]
You're right.

2017-03-13 10:46 GMT+01:00 Robert Forkel <notifications@github.com>:

**Pull request merged**